packages/admin/src/pages/scheme/[schemeId]/[exportId]/download-individual.page.tsx

```diff
@@ -0,0 +1,163 @@
+    );
+
+    availableSubmissionsTotalCount = submissionList.successCount;
+    exportedSubmissions = submissionList.exportedSubmissions;
+  } catch (err) {
+    return generateErrorPageRedirect(
+      'Something went wrong while trying to view submission applications.',
+      '/dashboard'
+    );
+  }
+
+  if (availableSubmissionsTotalCount === 0)
+    return generateErrorPageRedirect(
+      'There are no submissions available for download.',
+      '/dashboard'
+    );
+
+  return {
+    props: {
+      formAction: process.env.SUB_PATH + resolvedUrl,
+      schemeName: grantScheme.name,
+      availableSubmissionsTotalCount,
+      exportedSubmissions,
+      backBtnUrl: `/scheme/${schemeId}/${exportId}`,
+      csrfToken: res.getHeader('x-csrf-token') as string,
+    },
+  };
+};
+
+export const DownloadIndividualSubmissions = ({
+  formAction,
+  schemeName,
+  availableSubmissionsTotalCount,
+  exportedSubmissions,
+  backBtnUrl,
+  csrfToken,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const tableHeadColumns = [
+    {
+      name: <>{true && 'Name'}</>,
+      width: 'three-quarters',
+    },
+    {
+      name: 'Action',
+      isVisuallyHidden: true,
+    },
+  ] as TheadColumn[];
+
+  const tableRows = exportedSubmissions.map((submission) => {
+    return {
+      cells: [
+        {
+          content: <>{true && submission.name}</>,
+        },
+        {
+          content: (
+            <div className="govuk-!-text-align-right">
+              <CustomLink
+                href={`/apply/admin/api/signed-url?key=${encodeURIComponent(
+                  submission.zipFileLocation
+                )}`}
+                ariaLabel={`Download submission "${submission.name}"`}
+                excludeSubPath
+              >
+                Download
+              </CustomLink>
+            </div>
+          ),
+        },
+      ],
+    };
+  });
+
+  return (
+    <>
+      <Meta title={`Download applications - Manage a grant`} />
+      <CustomLink href={backBtnUrl} isBackButton />
+
+      <div className="govuk-!-padding-top-7">
+        <FlexibleQuestionPageLayout
+          fieldErrors={[]}
+          formAction={formAction}
+          csrfToken={csrfToken}
+        >
+          <span className="govuk-caption-l" data-cy="cyApplicationTitle">
+            {schemeName}
+          </span>
+          <h1 className="govuk-heading-l" tabIndex={-1}>
+            Download individual applications
+          </h1>
+
+          <div className="submissions-download-table">
+            <Table
+              tableClassName="table-thead-bottom-border"
+              caption="Submitted applications"
+              captionSize="m"
+              tHeadColumns={tableHeadColumns}
+              rows={tableRows}
+            />
+
+            <Pagination
+              additionalQueryData={{}}
+              itemsPerPage={10}
+              totalItems={availableSubmissionsTotalCount}
+              itemCountMargin={true}
```

Comment:
Can we maybe remove the additional margin above the 'showing 1 of 1" text when there is only one page of submissions?

<img width="724" alt="Screenshot 2024-02-29 at 14 44 51" src="https://github.com/cabinetoffice/gap-find-apply-web/assets/99667350/bc530d45-4815-4e5e-8d57-23d2689dd9d2">

Reply:
And instead of it saying Showing 1 of 1 items, it should be Showing 1 of 1 applications. In the pagination component you can pass this in through `itemType = 'applications'`

Reply:
Changed in latest push!